app/controllers/samples_controller.rb

```diff
@@ -155,8 +155,9 @@ def show
 
     @pipeline_run_display = curate_pipeline_run_display(@pipeline_run)
     @sample_status = @pipeline_run ? @pipeline_run.job_status : nil
-    @job_stats = @pipeline_run ? @pipeline_run.job_stats : nil
-    @summary_stats = @job_stats ? get_summary_stats(@job_stats) : nil
+    pipeline_run_id = @pipeline_run ? @pipeline_run.id : nil
+    job_stats_hash = make_job_stats_hash([pipeline_run_id])[pipeline_run_id]
```

Comment:
see other comment, this would be a call to job_stats_db_get

Reply:
ok